### PR TITLE
updating README.md

### PR DESCRIPTION
**Pull request by lezzokafka**
- Redirected "Local Full Node Development" , 
- Updated "Transfer CRO to another address".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -323,31 +124,47 @@ $ tendermint lite
 
 ## 8. Send Your First Transaction
 
-Genesis funds are bonded funds, to transfer freely around, you first have to withdraw to UTXO
-
-- Create Transfer Address
+ Before doing any transaction, It is noted that, the genesis fund is **bonded** at the beginning. To move funds freely around, we first have to withdraw it to UTXO:
```
Comment (tomtau) [inline README.md]: probably a bit more precise is that it's staked (some of the initial genesis amounts are bonded, some of them are unbonded)

File: README.md [hunk 2]
```diff
@@ -323,31 +124,47 @@ $ tendermint lite
 
 ## 8. Send Your First Transaction
```
Comment (tomtau) [inline README.md]: I guess it'll also make a sense to move this to the dev docs later

Comment (lezzokafka) [reply README.md]: Sure thing 👍

File: README.md [hunk 3]
```diff
@@ -323,31 +124,47 @@ $ tendermint lite
...
-Work in Progresss
+  ```bash
+  $ ./target/debug/client-cli transaction new --chain-id AB --name Default --type Withdraw
+  Enter passphrase: 
+  Enter staking address: 0xbdb46d64ed9da69093490a578158b1a20d96370b
+  Enter transfer address: dcro1pe7qg5gshrdl99m9q3ecpzvfr8zuk4h5qqgjyv6y24n80zye42as88x8tg
+  ```
 
+Transfer CRO to another address: 
+  - Once we have withdrawn the genesis funds, we can run the subcommand `sync`   to sync our wallet:
```
Comment (calvinaco) [inline README.md]: Extra space here?

File: README.md (hunk 1 above)
Comment (lezzokafka) [reply README.md]: Fixed, thanks Tomas

File: README.md [hunk 4]
```diff
@@ -323,31 +124,47 @@ $ tendermint lite
...
-Genesis funds are bonded funds, to transfer freely around, you first have to withdraw to UTXO
-
-- Create Transfer Address
+ Before doing any transaction, It is noted that, the genesis fund is **staked** at the beginning. To move funds freely around, we first have to withdraw it to UTXO:
 
-```bash
-$ ./target/debug/client-cli address new --name Default --type Transfer
-dcro1pe7qg5gshrdl99m9q3ecpzvfr8zuk4h5qqgjyv6y24n80zye42as88x8tg
-```
+- Create an address in type  `Transfer` to receive funds by:
```
Comment (calvinaco) [inline README.md]: Extra space here as well

File: README.md (hunk 3 above)
Comment (lezzokafka) [reply README.md]: Fixed, thanks Calvin

File: README.md (hunk 4 above)
Comment (lezzokafka) [reply README.md]: Fixed, thanks Calvin

**Pull request merged**